src/App.js

```diff
@@ -24,7 +24,10 @@ export default function App() {
   return (
     <>
       <Routes>
-        <Route path="/" element={<HomePage cards={cards} />} />
+        <Route
+          path="/"
+          element={<HomePage cards={cards} handleDelete={handleDeleteCard} />}
```

Comment:
Maybe you can choose the same name for the prop? :) 
`handleDeleteCard={handleDeleteCard}`